envoy/config/filter/http/health_check/v2/health_check.proto

```diff
@@ -20,7 +21,7 @@ message HealthCheck {
 
   // Specifies the incoming HTTP endpoint that should be considered the
   // health check endpoint. For example */healthcheck*.
-  string endpoint = 2 [(validate.rules).string.min_bytes = 1];
```

Comment:
I think we should keep the validation here, even though it's deprecated. Can you add both annotations?